### PR TITLE
Update page_head.html.twig

### PR DESCRIPTION
**Pull request by gospelpowered**
Added favicon output, if enabled in gantry, otherwise it will not render.

### CODE REVIEW

File: engines/common/nucleus/templates/page_head.html.twig [hunk 1]
```diff
@@ -17,6 +17,9 @@
     {%- block head_meta -%}
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        {% if gantry.config.styles.base.favicon %}
+        <link rel="icon" type="image/x-icon" href="{{ url(gantry.config.get('styles.base.favicon', false)) }}" />
```
Comment (mahagr) [inline engines/common/nucleus/templates/page_head.html.twig]: Can you remove default value false.. Maybe use the same code as one line above:

    <link rel="icon" type="image/x-icon" href="{{ url(gantry.config.styles.base.favicon) }}" />

File: engines/common/nucleus/templates/page_head.html.twig [hunk 2]
```diff
@@ -17,6 +17,9 @@
     {%- block head_meta -%}
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        {% if gantry.config.styles.base.favicon %}
+        <link rel="icon" type="image/x-icon" href="{{ url(gantry.config.styles.base.favicon) }}" />
```
Comment (gospelpowered) [inline engines/common/nucleus/templates/page_head.html.twig]: Changed the code to remove 'default' as requested.

**Comment by mahagr** [comment]
Looks good to me.

**Comment by w00fz** [comment]
Sweet! Thank you!

**Pull request merged**